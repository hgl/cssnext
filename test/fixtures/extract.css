:root {
	--width: 1px;
	--width: calc(1px + 1px);
	--width2: var(--width);
}
@custom-media --mq (max-width: 30em);
