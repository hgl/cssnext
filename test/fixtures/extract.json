{
  "features": {
    "customProperties": {
      "variables": {
        "color": "#e00",
        "color2": "var(--color)"
      }
    },
    "customMedia": {
      "extensions": {
        "mq2": "screen"
      }
    }
  }
}
