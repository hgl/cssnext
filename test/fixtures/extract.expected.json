{
	"customProperties":{
		"width": "2px",
		"width2": "2px",
		"color": "#e00",
		"color2": "#e00"
	},
	"customMedia":{
		"mq": "(max-width: 30em)",
		"mq2": "screen"
	}
}
